docs/index.md

```diff
@@ -31,6 +31,7 @@ api/exceptions
 api/plots/index
 api/utils
 api/stream_daq
+api/preprocessing
```

Comment:
I think the docs need a decent amount of reorganizing, so this is fine for now, but something like this should probably go in a "user guide" section rather than API docs. but since we don't have a proper user guide yet we can handle that when we do.